app/assets/stylesheets/application.css.scss

```diff
@@ -20,8 +20,29 @@ $neutral: #d9d9d9;
 
 @import "font-awesome";
 
-legend a {
-  font-weight: lighter;
+.main {
+  background-color: rgb(213, 203, 186);
+  padding-top: 20px;
+}
+
+legend {
+
+  a {
+    font-weight: lighter;
+  };
+}
+
+.search-form {
+
+  .button {
+    background-color: rgb(89, 84, 84);
+  };
+}
+
+.construction {
+  padding: 0.1px 10px 1px 10px;
```

Comment:
Properties should be ordered background-color, border-radius, padding<br>Shorthand form for property `padding` should be written more concisely as `0.1px 10px 1px` instead of `0.1px 10px 1px 10px`